src/cljc/rems/common/form.cljc

```diff
@@ -0,0 +1,24 @@
+(ns rems.common.form
```

Comment:
so rems.common is for cljc? should we rename the other cljc namespaces also? (not in this pr)

Reply:
I think it would make sense to think about this more. There is going to be some confusion here because there is rems.form in possibly db, api, application, editor and common variants. I made an issue of this https://github.com/CSCfi/rems/issues/1944